packages/nc-gui/components/template/Editor.vue

```diff
@@ -1025,7 +1028,7 @@ watch(modelRef, async () => {
   }
 
   :deep(.template-form-row) > td {
-    @apply p-0 mb-0;
+    @apply !p-1 mb-0 truncate max-w-50;
```

Comment:
why do we need `!` to override here?

Reply:
**fixed in below Commit**


[fix : removed padding important](https://github.com/nocodb/nocodb/pull/7221/commits/3b875d5bfc53b1c6bbe6b39f94fe592de808e5b3)
 
[3b875d5](https://github.com/nocodb/nocodb/pull/7221/commits/3b875d5bfc53b1c6bbe6b39f94fe592de808e5b3)